scripts/rdfxml2turtle.py

```diff
@@ -0,0 +1,12 @@
+from rdflib import Graph
```

Comment:
This solution is what we wanted to avoid in the first place. 

BTW, rdflib already comes with a command line`rdfpipe` doing exactly this. Check this CLI command. 

https://rdflib.readthedocs.io/en/stable/_modules/rdflib/tools/rdfpipe.html


Please use this, we do not need any python scripts developed.